src/modules/ens/ens.service.ts

```diff
@@ -194,8 +194,13 @@ export class EnsService implements OnModuleDestroy {
         return this.deleteNamespace(hash);
       }
 
-      const data = await this.domainReader.read({ node: hash });
       name = await this.domainReader.readName(hash);
+      // Don't want to sync roles or apps domains
```

Comment:
is this the agreed consensus? that we don't want to sync roles or apps? why though?

Reply:
also you need tests for this.

Reply:
I am working on tests right now.
We do not want to sync domains that starts with `apps` and `roles` such us: `roles.domain.iam.ewc` and `apps.domain.iam.ewc`. These domains do not have `metadata` key, but we are trying to read this key in the next line 
`const data = await this.domainReader.read({ node: hash });`.

This is related to this task https://energyweb.atlassian.net/jira/software/projects/ICS/boards/86?selectedIssue=ICS-52. But we are trying to catch these domains at `getAllNamespaces` method. But we were not catch these domains in ENS listeners.

Reply:
okay makes senses. thanks for clarifying.

Reply:
```suggestion
      // Don't want to sync domains that end in "roles" or "apps" (like "roles.domain.iam.ewc")
      // This is because these are "spacer" domains that do not have metadata
```
I think it